src/Controls/src/Core/Element/Element.cs

```diff
@@ -91,6 +91,8 @@ public string AutomationId
 			}
 		}
 
+		public bool IsPlatformViewNew { get; set; }
```

Comment:
I don't think there is a way to make this `internal` if it is a property on `IElement`.

Is there a way this could detect if `Handler` was `null` and became non-`null` for the first time? Then maybe we wouldn't need a new `bool` property?